deploy/kubernetes/examples/demo-snapshotclass-using-secret-from-configuration.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: snapshot.storage.k8s.io/v1beta1
```

Comment:
please align this filename to match the new storageclass filename (just with "snapshot" instead of "storage")

Reply:
it is aligned

Reply:
oh, you're right.
could we at least shorten them?
how about e.g. `demo-snapshotclass-config-secret.yaml`?

Reply:
done